js/widgets/widget-sortTbodies.js

```diff
@@ -38,6 +39,17 @@
 					// find parsers for each column
 					ts.sortTbodies.setTbodies( c, wo );
 					ts.updateCache( c, null, c.$tbodies );
+				})
+				.bind('sortEnd', function() {
+					// Moves the head row back to the top of the tbody
+					var lockHead = wo.sortTbody_lockHead;
+					var primaryRow = wo.sortTbody_primaryRow;
+
+					if ( primaryRow ) {
```

Comment:
Actually, since `lockHead` is only used once... change that line to:

```js
if ( wo.sortTbody_lockHead && primaryRow ) {
```

and remove the `var lockHead = wo.sortTbody_lockHead;`